service/modules/lin-reg-module/conda.yaml

```diff
@@ -0,0 +1,9 @@
+name: tutorial
+channels:
+  - defaults
+dependencies:
+  - pip:
+    - mlflow
+    - pandas
+    - diffprivlib
+    - git+https://git.repo/some_repo.git#egg=version_subpkg&subdirectory=repo
```

Comment:
This bug is inherited but we should probably resolve it before it spreads further. 

The line should be 

pip install git+https://github.com/privacytoolsproject/burdock.git#subdirectory=sdk


The line does not currently get called since most module executions remove the conda setup step